src/main.js

```diff
@@ -139,6 +139,18 @@ export class Draggable {
         };
     }
 
+    cancelDrag() {
+        unbind(this.document, "pointermove", this._pointermove);
+        unbind(this.document, "pointerup", this._pointerup);
+        unbind(this.document, "pointercancel", this._pointerup);
+
+        if (this.clickMoveClick) {
+            unbind(this.document, "contextmenu", preventDefault);
+        }
```

Comment:
`clickMoveClick` is not part of the `develop` branch, but rather added in https://github.com/telerik/kendo-draggable/pull/53/files#diff-1c3bdc38d306e81bc606bbd491192f6970a57e768208be1325afbb19ebddf578R7

which is currently on hold.

This means that this code should be removed.